liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -74,16 +74,17 @@ def __init__(self, edge_system_name, url, port, identity=None, tls_conf=None, qo
 
         self.client_id = client_id
 
+        if self.client_id is None or self.client_id == "":
```

Comment:
We should get consistent between None and ""; None makes sense for it not being set, and "" should be an invalid value.

Reply:
Fixed it. There was confusion previously if the unique client_id will be generated if value is set to "None", I verified it works.